channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1089,8 +1061,6 @@ static UINT rdpdr_process_connect(rdpdrPlugin* rdpdr)
 					WLog_ERR(TAG, "CreateThread failed!");
 					return ERROR_INTERNAL_ERROR;
 				}
-
-				continue;
```

Comment:
Why remove this? Adding a pseudo device for a hotplug wildcard does not really make sense or does it?

Reply:
The first_hotplug and hotplug thread on Windows exclude fixed drives but adding the speudo device redirects all my drives correctly.
Would it be better if check_path allowed more drive types?

Reply:
yes, might be the most simple solution. If that works for you too.